plugins/idea/pluginImpl/src/main/resources/META-INF/plugin.xml

```diff
@@ -45,7 +45,7 @@
     Milestones</a> on github.
     ]]>
     </change-notes>
-    <depends optional="true">org.intellij.scala</depends>
+    <!--<depends optional="true">org.intellij.scala</depends>-->
 
     <!-- please see http://confluence.jetbrains.com/display/IDEADEV/Build+Number+Ranges for description -->
     <idea-version since-build="139"/>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)